cli/src/psp-utils/buildCircom.ts

```diff
@@ -0,0 +1,229 @@
+import { createVerifyingkeyRsFile, toCamelCase } from "./index";
+import * as fs from "fs";
+import * as path from "path";
+import { randomBytes } from "tweetnacl";
+import { utils } from "@coral-xyz/anchor";
+import { downloadFile } from "./download";
+import { executeCommand } from "./process";
+import { executeAnchor, executeCircom } from "./toolchain";
+
+/**
+ * Generates a zk-SNARK circuit given a circuit name.
+ * Downloads the required powers of tau file if not available.
+ * Compiles the circuit, performs the groth16 setup, and exports the verification key.
+ * Cleans up temporary files upon completion.
+ * @param circuitName - The name of the circuit to be generated.
+ * @returns {Promise<void>}
+ */
+async function generateCircuit({
```

Comment:
can we unify this function different with generateCircuit in buildPsp?

Reply:
Yes, removed it from buildPsp, this is circom stuff, so I left it here.
buildPsp.ts file shrinked to one function: https://github.com/Lightprotocol/light-protocol/blob/sergey/prover/cli/src/psp-utils/buildPSP.ts